src/test/groovy/com/palantir/gradle/graal/GradleGraalEndToEndSpec.groovy

```diff
@@ -77,4 +78,103 @@ class GradleGraalEndToEndSpec extends IntegrationSpec {
         !result3.wasUpToDate(':nativeImage')
         output.getAbsolutePath().execute().text.equals("hello, world (modified)!\n")
     }
+
+    def 'allows specifying additional properties'() {
+        setup:
+        directory("src/main/java/com/palantir/test")
+        file("src/main/java/com/palantir/test/Main.java") << '''
+        package com.palantir.test;
+       
+        import java.io.IOException;
+        import java.net.URL;
+        
+        public final class Main {
+          public static final void main(String[] args) {
+            String result = null;
+            try {
+              result = convertStreamToString(new URL("http://www.google.com/").openStream());
+              System.out.println(result);
+            } catch (IOException e) {
+              throw new RuntimeException(e);
+            }
+        
+          }
+        
+          static String convertStreamToString(java.io.InputStream is) {
+            java.util.Scanner s = new java.util.Scanner(is).useDelimiter("\\\\A");
+            return s.hasNext() ? s.next() : "";
+          }
+        }
+        '''
+        buildFile << '''
+            apply plugin: 'com.palantir.graal'
+    
+            graal {
+                mainClass 'com.palantir.test.Main'
+                outputName 'hello-world'
+                graalVersion '1.0.0-rc5'
+                option '-H:EnableURLProtocols=http'
```

Comment:
why is this option `-H:EnableURLProtocols=http` instead of `--enable-url-protocols=http` as per https://github.com/oracle/graal/blob/master/substratevm/URL-PROTOCOLS.md ?

Reply:
Both work, when you run the app without the protocol  it suggest to add `-H:EnableURLProtocols=http`.